src/extension/package.json

```diff
@@ -143,10 +135,8 @@
     "ms-rest": "^2.5.4",
     "ms-rest-azure": "^3.0.0",
     "node-fetch": "^2.6.1",
-    "node-fs-extra": "^0.8.2",
     "opn": "^6.0.0",
     "p-retry": "^4.2.0",
-    "rimraf": "^3.0.2",
```

Comment:
I think fs-extra can replace that one https://www.npmjs.com/package/node-fs-extra